.github/workflows/scripts/run-build.sh

```diff
@@ -38,4 +23,4 @@ sudo -E chroot "$chroot" locale-gen
 sudo -E chroot "$chroot" useradd -m builduser
 sudo -E su -c 'echo "builduser ALL=NOPASSWD: ALL" >> "$chroot"/etc/sudoers'
 sudo -E chroot "$chroot" chown builduser -R /build
-sudo -E chroot "$chroot" su - builduser /bin/bash -c '/build/whistleblowing-software/.github/workflows/scripts/build_and_install.sh'
+sudo -E chroot "$chroot" su - builduser /bin/bash -c '/build/globaleaks-whistleblowing-software/.github/workflows/scripts/build_and_install.sh'
```

Comment:
This file has been edited across multiple commits with instances of code being removed and then re-added. To streamline the review, the offending commits should be removed, either by rebasing or squashing them into a single, meaningful commit.